src/Pages/UserPages/Landing/Landing.tsx

```diff
@@ -1,19 +1,292 @@
-import './Landing.module.scss'
-import { useEffect } from "react";
-import { useDispatch } from "react-redux";
+import { LandingImg } from '@/Assets/Images';
 import { fetchDataIslogged } from "@/Redux/Features/Auth/LoginSlice";
-
+import { AllAdsData } from '@/Redux/Features/Portal/Ads/getAllAdsSlice';
+import { Add, CalendarMonth, Favorite, Remove, Visibility } from '@mui/icons-material';
+import { Box, Button, Popover, TextField } from '@mui/material';
+import Typography from '@mui/material/Typography';
+import { DateRangeCalendar } from '@mui/x-date-pickers-pro/DateRangeCalendar';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
+import dayjs, { Dayjs, Range } from 'dayjs';
+import { useCallback, useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import './Landing.module.scss';
+import { AddFavoriteItem } from '@/Redux/Features/Portal/Favorites/AddToFavoriteSlice';
+import { toast } from 'react-toastify';
+import { RemoveFavoriteItem } from '@/Redux/Features/Portal/Favorites/RemoveFavoriteItemSlice';
+import { getFavoriteItems } from '@/Redux/Features/Portal/Favorites/GetAllFavoritesSlice';
+import { useNavigate } from 'react-router-dom';
 const Landing = () => {
 
   const dispatch = useDispatch();
+  const { count } = useSelector((state) => state.AddToFavorite)
+  const { data } = useSelector((state) => state.RemoveFavoriteItemSlice)
+
+  const [personsCount, setPersonsCount] = useState(1);
+const navigate =  useNavigate()
+
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [selectedDateRange, setSelectedDateRange] = useState<Range<Dayjs>>([
+    dayjs('2024-01-25'),
```

Comment:
We can change the default to the current day, and we should also prevent users from booking days in the past. As they are only allowed to book rooms in the present.